src/TunNetCom.SilkRoadErp.Sales.Api/Program.cs

```diff
@@ -6,7 +6,7 @@
 var builder = WebApplication.CreateBuilder(args);
 
 // Configure Serilog
-Log.Logger = new LoggerConfiguration()
+Serilog.Log.Logger = new LoggerConfiguration()
```

Comment:
Voila this is why "Log as a class is confusing, so choose for exemple LoggerMessageDefinitionsGen"